app/views/layouts/_footer.html.erb

```diff
@@ -0,0 +1,12 @@
+<footer class="footer">
+  <small>
+    The <a href="https://www.railstutorial.org/">Ruby on Rails Tutorial</a>
+by <a href="https://www.michaelhartl.com/">Michael Hartl</a> </small>
+    <nav>
+        <ul>
+            <li><%= link_to "About", about_path %></li>
```

Comment:
Yeah what I said about localised copy ... check out I18n or see if it happens later in the tutorial 🤔